.github/workflows/publish.yml

```diff
@@ -29,6 +32,12 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v2
 
+      - name: Install operator-courier
+        run: pip3 install operator-courier
```

Comment:
I did it with the ansible task in case anyone needed to test with a bug in an old(er) version of OpenShift. I don't think anyone has ever used it so maybe we can drop it.